index.ts

```diff
@@ -565,6 +585,8 @@ function tsdd(suite) {
         context.after = common.after;
         context.beforeEach = common.beforeEach;
         context.afterEach = common.afterEach;
+
+        /* istanbul ignore next */
```

Comment:
I believe that we should add a test that actually covers this. Otherwise, this is unused code that should be eliminated.

Reply:
This is copied from mocha :( I am not sure how this is used.
On the other hand the best approach here would be to run all mocha TDD and BDD tests using the mocha-typescript interface. But that's a little bit too much for me right now.